modules/user-guide/pages/gitlab-code-quality-reports.adoc

```diff
@@ -0,0 +1,16 @@
+// Metadata for Antora
+:navtitle: GitLab Code Quality reports
+:keywords: vale gitlab
+// :page-aliases: 
+// End of metadata for Antora
+
+:parent-context-of-enabling-style-compliance-reports-in-gitlab-merge-requests: {context}
+
+[id="configuring-vale-for-gitlab-code-quality-reports_{context}"]
+= GitLab Code Quality reports
+
+:context: enabling-style-compliance-reports-in-gitlab-merge-requests
+
+include::partial$proc_enabling-style-compliance-reports-in-gitlab-merge-requests.adoc[leveloffset=+1]
```

Comment:
As it contains only one procedure, you could add directly the content of the procedure into the page.

Reply:
We may need a verification procedure of some sort in the future. Will leave for now but open an issue to track it.